packages/rrweb/test/integration.test.ts

```diff
@@ -209,6 +209,55 @@ describe('record integration tests', function (this: ISuite) {
     assertSnapshot(snapshots);
   });
 
+  it('can record style changes compactly and preserve css var() functions', async () => {
+    const page: puppeteer.Page = await browser.newPage();
+    await page.goto('about:blank');
+    await page.setContent(getHtml.call(this, 'blank.html'), {
+      waitUntil: 'networkidle0',
+    });
+
+    // goal here is to ensure var(--mystery) ends up in the mutations (CSSOM fails in this case)
+    await page.evaluate(
+      'document.body.setAttribute("style", "background: var(--mystery)")',
+    );
+    await page.waitForTimeout(10);
```

Comment:
Wait for requestAnimationFrame is more reliable than using timeout
```suggestion
    await waitForRAF(page);
```